mod/da/pkg/da/service.go

```diff
@@ -41,6 +44,7 @@ type Service[
 		AvailabilityStoreT, BeaconBlockBodyT,
 		BlobSidecarsT, ExecutionPayloadT,
 	]
+	feed   *event.FeedOf[asynctypes.EventID, *asynctypes.Event[BlobSidecarsT]]
```

Comment:
nit: `sidecarFeed` for clarity and maintaining consistent naming with val/chain services